src/conf.rs

```diff
@@ -94,15 +96,48 @@ pub fn load(r: &mut Read, o: PathBuf) -> Result<Config, ConfigError> {
                     }),
                     _ => String::from("/usr/bin/sh"),
                 },
+                output: match v.get("output") {
+                    Some(&toml::Value::String(ref s)) => s.clone(),
+                    Some(_) => return Err(ConfigError {
+                        kind: ConfigErrorKind::MismatchedShellType,
```

Comment:
Please create a new enum type and rename it to  `OutputType` like the shell one.